x-pack/plugins/security_solution/common/endpoint/types.ts

```diff
@@ -253,7 +253,11 @@ export type AlertEvent = Immutable<{
   };
   endpoint: {
     policy: {
-      id: string;
+      applied: {
```

Comment:
hmm I don't think alerts will have this structure. The mapping for alerts looks like:

```
endpoint:
  fields:
    policy: 
      fields:
        id: {}
    artifact: {}
```
based on here: https://github.com/elastic/endpoint-package/blob/master/custom_subsets/elastic_endpoint/events/malware_event.yaml#L26

Do we need to update the mapping?

Reply:
Yes

Reply:
https://github.com/elastic/endpoint-package/pull/10